dashboard/app/models/resource.rb

```diff
@@ -95,6 +95,9 @@ def summarize_for_lesson_plan
       key: key,
       name: get_localized_property(:name),
       url: get_localized_property(:url),
+      # used by lesson materials page
+      downloadUrl: download_url,
```

Comment:
Are `downloadUrl` and `download_url` going to be the same value? If so, are we sure we want to add the complexity here or should we just parse it on the front-end

Reply:
it turned out to be a pain to just parse this on the frontend, because it would require duplicating a bunch of type definitions (e.g. Resource and ServerResource, Lesson and ServerLesson, LessonMaterialsData and ServerLessonMaterialsData) or else updating the Resource type to have `downloadUrl` and `download_url` as both optional. this seemed like the simplest option to me, and, sending params to the client in camel case was already a best practices before the introduction of typescript problems IIRC. I'd be happy to take a follow-up to move other uses from download_url to downloadUrl